src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerTextView.cs

```diff
@@ -364,6 +364,25 @@ public void DisconnectFromIntellisenseControllers()
             }
         }
 
+        public void QueuePostLayoutAction(Action action) => _innerTextView.QueuePostLayoutAction(action);
```

Comment:
These are all new ITextView2 methods?

Reply:
Yes

Reply:
yes

---
In reply to: [223910652](https://github.com/dotnet/roslyn/pull/30419#discussion_r223910652) [](ancestors = 223910652)